src/main/java/cc/hyperium/commands/defaults/NameHistoryCommand.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Hyperium Client, Free client with huds and popular mod
+ * Copyright (C) 2018  Hyperium Dev Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published
+ *  by the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package cc.hyperium.commands.defaults;
+
+import cc.hyperium.commands.BaseCommand;
+import cc.hyperium.commands.CommandException;
+import cc.hyperium.gui.NameHistoryGui;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NameHistoryCommand implements BaseCommand {
```

Comment:
This command needs to be registered, along with all of the others in `Hyperium#registerCommands`.